docs/current_docs/manuals/developer/cookbook.mdx

```diff
@@ -97,26 +97,26 @@ dagger call modify-directory \
 
 ## Secrets
 
-### Use secrets
+### Use secret variables
```

Comment:
```suggestion
### Use host variables, files, or commands as secrets
```

Reply:
This also feels targeted towards what the CLI example is doing rather than what the code is doing

Reply:
I think you're right. Ignore this suggestion but I also don't think the original version is very clear. WDYT of `Use variables as secrets`?